Packs/GitHub/Integrations/GitHub/GitHub.py

```diff
@@ -1323,23 +1323,30 @@ def get_workflow_usage(owner_name, repository_name, workflow_id):
     return workflow_usage
 
 
-def get_team_members_command():
+def list_team_members_command():
     args = demisto.args()
     org = args.get('organization')
     team_slug = args.get('team_slug')
-    response = get_team_members(org, team_slug)
+    limit = int(args.get('limit'))
+    response = get_team_members(org, team_slug, limit)
     members = []
     for member in response:
-        ec_object = {
+        context_data = {
             'ID': member.get("id"),
-            'Login': member.get("login"),
+            'Login': member.get("login")
         }
-        members.append(ec_object)
-    ec = {
-        'GitHub.TEAMMEMBER(val.ID === obj.ID)': members
-    }
-    human_readable = tableToMarkdown(f'Team Member of team {team_slug} in organization {org}', t=members, removeNull=True)
-    return_outputs(readable_output=human_readable, outputs=ec, raw_response=response)
+        members.append(context_data)
+    if members:
+        human_readable = tableToMarkdown(f'Team Member of team {team_slug} in organization {org}', t=members, removeNull=True)
+    else:
+        human_readable = f'There is no team members on team {team_slug} in organization {org}'
+
+    return_results(CommandResults(
+        readable_output=human_readable,
+        outputs_prefix=f'GitHub.TEAMMEMBER.{team_slug}(val.ID === obj.ID)',
```

Comment:
What happens when you run the same command twice, and in between one or more of the users have been removed from it?

Reply:
They will stay in the context.. we could not delete them